code/modules/customitems/item_defines.dm

```diff
@@ -426,6 +426,73 @@
 
 //////////// Masks ////////////
 
+
+obj/item/clothing/mask/gas/sechailer/fluff/tyhailer // T0EPIC4U: Ty Omaha
+	name = "tacticool security gas mask"
+	desc = "A tacticool Security gas mask with modified 'Compli-o-nator 3000' device, it's labeled: \"Ty Omaha\". It looks like it has some..residue..you probably don't want to touch it."
+	icon_state = "sechailer"
+	var/phrase = 1
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust)
+
+/obj/item/clothing/mask/gas/sechailer/fluff/tyhailer/ui_action_click(mob/user, actiontype)
+	if(actiontype == /datum/action/item_action/halt)
+		tyhalt()
+	else
+		switch(phrase)
+			if(1)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Stop or Bash</span>")
+				phrase = 2
+			if(2)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Criminal Scum</span>")
+				phrase = 3
+			if(3)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Prepare Justice</span>")
+				phrase = 4
+			if(4)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Shut Up</span>")
+				phrase = 5
+			if(5)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: The Law</span>")
+				phrase = 6
+			if(6)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Dead or Alive</span>")
+				phrase = 7
+			if(7)
+				to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Golden Bolt</span>")
+				phrase = 1
+
+/obj/item/clothing/mask/gas/sechailer/fluff/tyhailer/proc/tyhalt()
```

Comment:
Dunno what you mean by it will use the regular sechailer's halt, I tested it and everything works